cmd/gapit/sxs_video.go

```diff
@@ -107,7 +107,16 @@ func (verb *videoVerb) sxsVideoSource(
 	// be anything and the replayed frame will show the undefined framebuffer
 	// pattern.
 	// Permit the first run of frames to have no content.
```

Comment:
Clarify the initialization logic in the comment:
Permit the first run of frames to have no content, but only if we are sure there is eventually going to be some Clear or Drawcall